app/Listeners/RunForSavedForm.php

```diff
@@ -0,0 +1,21 @@
+<?php namespace App\Listeners;
+
+use Anomaly\SettingsModule\Setting\Form\SettingFormRepository;
+use Anomaly\Streams\Platform\Ui\Form\Event\FormWasSaved;
+
+class RunForSavedForm
```

Comment:
This name is general, the listener is only handling the maintenance mode change, you can name it EnableMaintenanceMode for example